docs/source/tutorial/interfaces.rst

```diff
@@ -404,14 +402,14 @@ case back as follows:
 The effect of this version of ``readNumbers`` is identical to the first (in
 fact, it is syntactic sugar for it and directly translated back into that form).
 The first part of each statement (``Just x_ok <-`` and ``Just y_ok <-``) gives
-the preferred binding - if this matches, execution will continue with the rest
+the preferred binding — if this matches, execution will continue with the rest
 of the ``do`` block. The second part gives the alternative bindings, of which
 there may be more than one.
 
-``!``-notation
+``!`` notation
```

Comment:
Should be reverted throughout; same comment as "`do`-notation". Apologies.

Reply:
Done.